spacelift/data_saved_filters_test.go

```diff
@@ -0,0 +1,40 @@
+package spacelift
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	. "github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal/testhelpers"
+)
+
+func TestFiltersData(t *testing.T) {
+	t.Run("load all saved filters", func(t *testing.T) {
+		randomID := acctest.RandStringFromCharSet(5, acctest.CharSetAlphaNum)
+
+		datasourceName := "data.spacelift_saved_filters.all"
+
+		testSteps(t, []resource.TestStep{{
```

Comment:
I think we could do with a few more tests here - testing for each filter, and also for no results maybe?

Reply:
Added more tests.